tests/identity/id.test.ts

```diff
@@ -167,4 +168,157 @@ describe('identity', () => {
 
     await credWallet.getRevocationStatusFromCredential(issuerCred);
   });
+
+  it('createIdentity Secp256k1', async () => {
+    const ethSigner = new Wallet(WALLET_KEY, (dataStorage.states as EthStateStorage).provider);
+
+    const { did, credential } = await createEthereumBasedIdentity(idWallet, {
+      ethereumBasedIdentityOpts: {
```

Comment:
This was proposed here https://github.com/0xPolygonID/js-sdk/pull/200#discussion_r1553169009. We could also pass the other optional parameter although it's not passed here and we get the default value.
```
   ethereumBasedIdentityOpts: {
        ethSigner: Signer;
        createBjjCredential?: boolean;
    }
```

Reply:
@daveroga previous comment  was actual, because we had one function createIdentity.

now as we have two separate functions it makes sense to remove one level of nested structures and just have options for createEthereumBasedIdentity (EthereumBasedIdentityCreationOptions)  and remove field ethereumBasedIdentityOpts

Reply:
Yes. You are right. Updated.